validation-frameworks/functional/README.md

```diff
@@ -63,14 +63,23 @@ def test_check_logs():
 ### Running Functional Tests
 
 Follow these steps to run the functional tests:
-
-1. Navigate to the functional testing directory:
+1. Fetch kubeconfig for fleet-hub-cluster. Update the region in the below command accordingly
+```
+aws eks --region us-east-1 update-kubeconfig --name fleet-hub-cluster
```

Comment:
- Do not assume the region is `us-east-1` assume there is a variable `AWS_REGION` set.
- Validation should run on the spoke clusters not on the control plane (hub) cluster. Use cluster name `fleet-spoke-staging`
```suggestion
aws eks --region $AWS_REGION update-kubeconfig --name fleet-spoke-staging
```